src/graphql/commitWithProjectIdFingerprint.graphql

```diff
@@ -0,0 +1,9 @@
+subscription CommitWithProjectIdFingerprint {
```

Comment:
This commit does not seem to have a ProjectIdFingerprint.

Reply:
Good catch. Yeah, I removed it. At first I was getting it so that I could see if it was already there somehow. But then I reconsidered and simplified it. I should change the name to reflect that.